multimedia/icecast/files/icecast.init

```diff
@@ -1,12 +1,21 @@
 #!/bin/sh /etc/rc.common
-# Example script
-# Copyright (C) 2014 OpenWrt.org
+
+# Startup script
+# Copyright (C) 2015 OpenWrt.org
  
 USE_PROCD=1
-START=99
+START=90
 STOP=15
 
 start_service() {
+	user_exists icecast 87 || user_add icecast 87
+	group_exists icecast 87 || group_add icecast 87
```

Comment:
There is a USERID statement in the Makefile too. Adding these lines to the init handles the 'sysupgrade' case when the passwd and group files are preserved.